packages/database/src/migrations/20210422030246-CreateUnknownEntitiesForSamoa-modifies-data.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import { insertObject } from '../utilities';
+import { generateId } from '../utilities/generateId';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const NEW_UNKNOWN_DISTRICT_ENTITY = {
+  id: generateId(),
+  code: 'WS_Unknown_District',
+  parent_id: '5df1b88c61f76a485cd1ca09',
+  name: 'Unknown District',
+  type: 'district',
+  country_code: 'WS',
+  metadata: '{"dhis": {"isDataRegional": true}}',
+  attributes: '{}',
+};
+
+const NEW_UNKNOWN_FACILITY_ENTITY = {
+  id: generateId(),
+  code: 'WS_Unknown_Facility',
+  parent_id: `${NEW_UNKNOWN_DISTRICT_ENTITY.id}`,
+  name: 'Unknown Facility',
+  type: 'facility',
+  country_code: 'WS',
+  metadata: '{"dhis": {"isDataRegional": true}}',
+  attributes: '{}',
+};
+
+const NEW_UNKNOWN_VILLAGE_ENTITY = {
+  id: generateId(),
+  code: 'WS_Unknown_Village',
+  parent_id: `${NEW_UNKNOWN_FACILITY_ENTITY.id}`,
+  name: 'Unknown Village',
+  type: 'village',
+  country_code: 'WS',
+  metadata: '{"dhis": {"isDataRegional": true}}',
+  attributes: '{}',
+};
+
+const ENTITY_RELATION_UPOLU_EXPLORE = {
+  id: generateId(),
+  parent_id: '5df1b88c61f76a485cd1ca09',
```

Comment:
I think I've said this in an earlier PR review for you - please don't just have ids with no explanation.

Ideal way: Add an extra query that fetches the id based on the entity code

Not so ideal way: Hard code the id with a comment indicating the code it relates to

Not allowed way: Just the id

Would be great to do this in the ideal way - search `codeToId` in `migrations` for examples

![Screen Shot 2021-04-27 at 9 43 15 AM](https://user-images.githubusercontent.com/1274422/116154660-0f82b500-a73d-11eb-989c-df712acebc28.png)